OutOfSchool/OutOfSchool.BusinessLogic/Services/ChildService.cs

```diff
@@ -539,7 +541,7 @@ private async Task CompleteChildChangesAsync()
     {
         try
         {
-            await childRepository.UnitOfWork.CompleteAsync();
+            await childRepository.SaveChangesAsync();
```

Comment:
We have to make a choice whether to use or not `ConfigureAwait` bring it to a single form.
I suggest to use them everywhere where it is possible. But we can have a discussion about it.